docker/bookkeeper/entrypoint.sh

```diff
@@ -61,14 +63,14 @@ echo "wait for zookeeper"
 until ${ZK_HOME}/bin/zkCli.sh -server $ZK_URL ls /; do sleep 2; done
 
 echo "create the zk root"
-${ZK_HOME}/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH}
-${ZK_HOME}/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}
-${ZK_HOME}/bin/zkCli.sh -server $ZK_URL create /${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}/${BK_CLUSTER_NAME}
+# Silence exit codes with "|| :" as the commands can safely fail with a "Node already exists" error
```

Comment:
This is dangerous as only the specific error needs to be ignored. We should continue failing in case of all the other errors.

Reply:
The problem is that `zkCli.sh` does not return specific error codes based on the error cause. Realistically, the command should not throw any other error in this scenario. The connection to the ZK server is checked right before those lines, and the only potential error is that the path already exists... In any case, this is an improvement compared to the previous script version, where all commands could fail silently.

Reply:
I agree that this is an improvement over previous status. Please spend some time to figure out a way to take care of this better. If there is no easy way, let's log an issue to track this.

Reply:
@arvindkandhare I'll create a new issue to track this. But I would like to have this PR merged first so that I can include proper reference to the script commands.

Reply:
Post the issue number here, please.

Reply:
https://github.com/pravega/pravega/issues/3039